Examples/Conference/SwiftUI/Shared/Screens/Chat/ChatMessageStore.swift

```diff
@@ -49,7 +49,7 @@ final class ChatMessageStore: ObservableObject {
             return false
         }
 
-        return try await !chat.sendMessage(text)
+        return try await chat.sendMessage(text)
```

Comment:
What was the reason for flipping the return value here?

Reply:
It was a bug in the chat view, the error icon was shown when the message was successfully sent.

Reply:
Thanks for clarification!